### ISSUE TITLE
Merge staging process into the charter proposal

### ISSUE THREAD

**Issue by evanp** [swicg/potential-charters#36]
We haven't approved the main CG charter yet. Structuring the staging process as an amendment to the charter is kind of a big hassle. Could we just include it in the main CG charter document and just approve the whole thing at the same time?

**Comment by gobengo** [comment]
> Structuring the staging process as an amendment to the charter is kind of a big hassle.

Is it? I think this could just be a PR onto the charter and a proposal for consensus to amend charter?

I think it might be more of a hassle to reach full consensus on a staging process in an initial charter. Maybe something simpler and more in line with the decision process that's already in the charter? or, if deviating from that, something that several other CGs have found useful to adopt (a la [decision policies with calls for consensus on list as we discussed a while ago](https://lists.w3.org/Archives/Public/public-swicg/2023Sep/0118.html))

Personally, my gut is that this issue is not a top priority, and it could instead make sense to prioritize:
1. reach consensus on any CG charter w/ decision process
2. unblock CG work on deciding a WG Charter, which I think doesn't require staging process
3. work on staging process document that includes text on what it would amend the charter decision policy to (e.g. 'this group will not decide to publish documents that have not reached stage 2 on this process')
4. amend charter, only if work on staging process document deems it's necessary. it may not. 

Anyway, just sharing those thoughts async. 

> Could we just include it in the main CG charter document and just approve the whole thing at the same time?

As the author of #1, I'd be willing to review a PR adding this and otherwise talk more about it, but it's omission wouldn't be a hard blocker for me. Is the reason you're raising this issue that you would block consensus with the potential charter's current decision policy?

**Comment by evanp** [comment]
> > Structuring the staging process as an amendment to the charter is kind of a big hassle.
> 
> Is it? I think this could just be a PR onto the charter and a proposal for consensus to amend charter?

Yes. "A big hassle" is hyperbole. There are a lot bigger hassles in life. I think approving a document when we know that an amendment is coming, and then making the amendment, is incrementally harder than amending the document before you approve it, and approving the whole package once.

I'm not sure how we're going to decide on approving the charter -- see #45 -- but if it's less onerous than the amendment process (30-day vote, 2/3 supermajority), maybe it's easier to do it all as a package.

**Comment by evanp** [comment]
> Personally, my gut is that this issue is not a top priority

That's OK! Feel free to ignore issues that are not a priority to you personally. Please don't presume to tell me what my priorities are, though.

**Comment by evanp** [comment]
Per discussion at CG meeting, we seem to be in consensus to not do this. So, I'm going to close this issue.
